src/Console/EnvHelperTrait.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace PHPOpenSourceSaver\JWTAuth\Console;
+
+use Closure;
+use Illuminate\Support\Str;
+
+trait EnvHelperTrait
+{
+    /**
+     * Checks if the env file exists
+     * 
+     * @return bool
+     */
+    function envFileExists(): bool
+    {
+        return file_exists($this->envPath());
+    }
+
+    /**
+     * Update an env-file entry
+     * 
+     * @param string $key
+     * @param string|int $value
+     * @param Closure|null $confirmOnExisting
+     * @return bool
+     */
+    function updateEnvEntry(string $key, $value, Closure $confirmOnExisting = null): bool
+    {
+        static $filepath = null;
+        
+        if(is_null($filepath)) {
+            $filepath = $this->envPath();
+        }
```

Comment:
Is there a benefit to the `static`? I think we can just call `$this->envPath()` all the time?

Reply:
I have changed this in https://github.com/PHP-Open-Source-Saver/jwt-auth/pull/145